docs/jailer.md

```diff
@@ -1,5 +1,9 @@
 # The Firecracker Jailer
 
+## Disclaimer
+
+The jailer is designed to work only with statically linked binaries (with the default musl toolchain) and will not work with experimental gnu builds.
```

Comment:
All lines in the .md files should be 80 chars max.

Reply:
will update. is there a place where these guidelines are mentioned?